clients/src/test/java/org/apache/kafka/common/requests/TxnOffsetCommitRequestTest.java

```diff
@@ -80,10 +81,10 @@ public void setUp() {
         );
     }
 
-    @Test
     @Override
-    public void testConstructor() {
-
+    @ParameterizedTest
+    @ApiKeyVersionsSource(apiKey = ApiKeys.TXN_OFFSET_COMMIT)
+    public void testConstructor(short version) {
```

Comment:
Is this change related to the PR? If not, I would rather do it in a separate PR.